app/services/migration/migrators/application.rb

```diff
@@ -0,0 +1,48 @@
+module Migration::Migrators
+  class Application < Base
+    def call
+      migrate(ecf_npq_applications, :application) do |ecf_npq_application|
+        application = ::Application.find_by!(ecf_id: ecf_npq_application.id)
+
+        compare_attributes_values!(ecf_npq_application, application)
+      end
+    end
+
+  private
+
+    def ecf_npq_applications
+      @ecf_npq_applications ||= Migration::Ecf::NpqApplication.joins(:participant_identity).all.limit(500)
+    end
+
+    def attributes_to_compare
+      %i[
```

Comment:
As we're doing this a lot of times it would be more performant to move these to a frozen constant